pages/home.html

```diff
@@ -2,9 +2,31 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="../css/bootstrap.css">
+    <link rel="stylesheet" href="../css/master.css">
     <title>Home Page</title>
   </head>
-  <body>
+  <body id=homeBackground>
```

Comment:
Instead of putting an id on the body element, try using a `<main>` tag with the same id.

Reply:
will do

Reply:
done